src/services/guidanceresponse.service.ts

```diff
@@ -8,7 +44,30 @@ module.exports.create = async (args: any, req: any) => {
 
 
 module.exports.remsEtasu = async (args: any, context: any, logger: any) => {
+    const parameters: Parameters = args?.resource;
+    let medication: Medication | MedicationRequest | undefined;
```

Comment:
Can we add a log here so we can show that the request is coming in? We do this on the CDS Hooks side and it would be nice here as well.